packages/transaction_signing_gateway/lib/encrypt/aes_cipher.dart

```diff
@@ -0,0 +1,56 @@
+import 'package:encrypt/encrypt.dart';
+import 'package:transaction_signing_gateway/encrypt/cipher.dart';
+
+class AESCipher implements Cipher {
+  @override
+  String decrypt({required String password, required String encryptedData}) {
+    final encrypter = _fromPassword(password);
```

Comment:
`encryptedData` consists of IV and encrypted message, we split that up with `_encryptedBundle.fromString(...)` and use the data to perform decryption